experimental/packages/opentelemetry-sdk-web/README.md

```diff
@@ -0,0 +1,122 @@
+# OpenTelemetry SDK for Web
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This package provides the full OpenTelemetry SDK for Web including tracing.
+
+## Quick Start
+
+To get started you need to install `@opentelemetry/sdk-web`, a tracing exporter, and any appropriate instrumentation for the web modules used by your application.
+
+### Installation
+
+```sh
+$ # Install the SDK
+$ npm install @opentelemetry/sdk-web
+
+$ # Install exporters
+$ npm install @opentelemetry/exporter-trace-otlp-http
+
+$ # or install all officially supported core and contrib plugins
+$ npm install @opentelemetry/auto-instrumentations-web
+
+```
+
+### Initialize the SDK
+
+Before any other module in your application is loaded, you must initialize the SDK.
+If you fail to initialize the SDK or initialize it too late, no-op implementations will be provided to any library which acquires a tracer or meter from the API.
+
+This example uses the OTLP Http Json exporter, but the OTLP Proto exporter can be used as well.
+As shown in the installation instructions, exporters passed to the SDK must be installed alongside `@opentelemetry/sdk-web`.
+
+```javascript
+const opentelemetry = require("@opentelemetry/sdk-web");
+cont { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-http');
+const {
+  getWebAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-web");
+
+const collectorOptions = {
+  url: '<opentelemetry-collector-url>', // url is optional and can be omitted - default is http://localhost:4318/v1/traces
+  headers: {}, // an optional object containing custom headers to be sent with each request
+  concurrencyLimit: 10, // an optional limit on pending requests
+};
+
+const otlpTraceExporter = new OTLPTraceExporter(collectorOptions);
+
+const sdk = new opentelemetry.WebSDK({
+  // Optional - if omitted, the tracing SDK will be initialized from environment variables
```

Comment:
environment variables do not apply, so this may not really be optional.
Can we also add a log/event Exporter as well here?

Reply:
Removed